src/api/IBMiContent.ts

```
@@ -631,6 +631,45 @@ export default class IBMiContent {
     }
   }
 
+  /**
+   *
+   * @param filter: the criterias used to list the members
+   * @returns
+   */
+  async getMemberInfo(library: string, sourceFile: string, member: string): Promise<IBMiMember | undefined> {
```

I think throwing `undefined` if the member does not exist makes sense. But, I think this API should throw an error if the feature (`GETMBRINFO.SQL`) is not installed.